### PR TITLE
Initialized bun project with TypeScript configuration and pnpm monorepo setup (awaiting review for completion)

### PR DESCRIPTION
**Pull request by s29papi**
awaiting review for completion.

**Tasks done:**
 
1. Set up the monorepo with a app/extension package and a packages/light-client package.

2. The app/extension package should be importing something from the packages/light-client package

sol addy: 8fFsLKGv5sizL97mjtrVvPu9trQAHZx5HBeLGzLVAt2o

### CODE REVIEW

**Comment by roninjin10** [comment]
Nice start! Check out tevm monorepo for example and let me know if you need any help

**Comment by s29papi** [comment]
Hello @roninjin10, I would appreciate a pr review.

File: app/extension/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"name": "@onchain-clippy/extension",
+	"version": "1.0.0",
+	"private": false,
+	"sideEffects": false,
+	"type": "module",
+
```
Comment (roninjin10) [inline app/extension/package.json]:
```suggestion
```

File: app/extension/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	"scripts": {
+		"all": "pnpm i"
+	},
+	"keywords": [],
+	"author": "",
+	"license": "ISC",
```
Comment (roninjin10) [inline app/extension/package.json]:
```suggestion
	"license": "MIT",
```

File: app/extension/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	"dependencies": {
+		"@onchain-clippy/light-client": "workspace:^",
+		"@onchain-clippy/tsconfig": "workspace:^",
+		"@onchain-clippy/tsupconfig": "workspace:^"
```
Comment (roninjin10) [inline app/extension/package.json]: Let's remove these packages and instead just install `@tevm/tsconfig` and `@tevm/tsupconfig`

File: configs/tsconfig/package.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (roninjin10) [inline configs/tsconfig/package.json]: Let's remove the configs folder and just install from `@tevm/tsconfing

File: configs/tsupconfig/dist/index.cjs [hunk 5]
```diff
@@ -0,0 +1,62 @@
+'use strict';
```
Comment (roninjin10) [inline configs/tsupconfig/dist/index.cjs]: Same let's install from `@tevm/tsupconfig`

File: packages/light-client/dist/index.d.cts [hunk 6]
```diff
@@ -0,0 +1,12 @@
+type DummyLightClient = {
+    /**
+     * The url being used to connect to the remote Tevm backend
+     */
+    url: string;
+    /**
+     * Name of the client
+     */
+    name: string;
+};
```
Comment (roninjin10) [inline packages/light-client/dist/index.d.cts]: Remove all the dist/ files

**Review by roninjin10 — CHANGES_REQUESTED** [review]
Great start!

File: packages/light-client/dist/index.d.cts (hunk 6 above)
Comment (s29papi) [reply packages/light-client/dist/index.d.cts]: done this through gitignore, should i totally prevent it from building through the package json or does this work ?

**Comment by roninjin10** [comment]
Thanks for the contribution and providing address! I will send a holiday gift of appreciation

**Pull request merged**